COLLECTION.md

```diff
@@ -6,6 +6,7 @@
 
 |  #  | Brand              | Type        | Name                        | Origin      | Flavour                     |
 |-----|--------------------|-------------|-----------------------------|-------------|-----------------------------|
+| 142 | [Jia ming]         | [Green]     | [West Lake Dragon Well]     | [China]     | Green tea                   |
```

Comment:
Well, I do not see the problem in links. Where I do see the problem is adding link that does not exist like `[Jia ming]` above. Why are you adding link that does not exist?